### PR TITLE
fix: adds accessibility/automation attributes

### PR DESCRIPTION
**Pull request by ritikavnair**
PR checklist:

- [x] Github Issue associated with branch.
- [x] Code passes eslint analysis.
- [ ] Code passes cross browser testing.
- [ ] Test code coverage is 85% or greater.
- [x] PR branch is up-to-date with latest master branch.

Solving #79

### CODE REVIEW

**Comment by ritikavnair** [comment]
@MadaraUchiha-314 Hi, just trying to take a dig at this issue, is this what is expected for accessibility/automation attributes? Please review, thanks!

File: src/FormattedMessage.jsx [hunk 1]
```diff
@@ -24,7 +24,7 @@ const FormattedMessage = (props) => (
         fontSize: 'inherit',
       };
       /* eslint-disable react/no-danger */
-      return <span style={style} dangerouslySetInnerHTML={{ __html: messageString }} />;
+      return <span automation-id="Formatted-Message" aria-label="formatted text" style={style} dangerouslySetInnerHTML={{ __html: messageString }} />;
```
Comment (MadaraUchiha-314) [inline src/FormattedMessage.jsx:27]: The aria-label should be the text itself. The automation id I feel should be the key itself.

Also, these should be the defaults, We should give users the option to override.

Comment (ritikavnair) [reply src/FormattedMessage.jsx]: @MadaraUchiha-314  Okay, got it. Updated to pass in a custom aria label and also gave it a default value. Changed automation-id. Let me know what you think.

File: src/FormattedMessage.jsx [hunk 2]
```diff
@@ -23,8 +25,16 @@ const FormattedMessage = (props) => (
       const style = {
         fontSize: 'inherit',
       };
+
       /* eslint-disable react/no-danger */
-      return <span style={style} dangerouslySetInnerHTML={{ __html: messageString }} />;
+      return (
+        <span
+          automation-id={id}
```
Comment (MadaraUchiha-314) [inline src/FormattedMessage.jsx:32]: Need the same ability for automation-id as well, if users are specifying an automation id explicitly, then we need to use that.

Comment (ritikavnair) [reply src/FormattedMessage.jsx]: @MadaraUchiha-314 Updated to allow custom automation id. Let me know if it looks good, thanks.

**Comment by ritikavnair** [comment]
@MadaraUchiha-314  If no other changes required and PR approved, can you please add the hacktoberfest-accepted label ? Thanks!

**Review by MadaraUchiha-314 — APPROVED** [review]

**Comment by roxiomontes** [comment]
@MadaraUchiha-314  will you merge the PR?

**Pull request closed without merging**